junebug/stores.py

```diff
@@ -21,6 +22,11 @@ def _redis_op(self, func, id, *args, **kwargs):
         yield self.redis.expire(id, self.ttl)
         returnValue(val)
 
+    def get_key(self, channel_id, message_id):
+        '''Gets the key where a message would be stored, using the channel and
+        message ids'''
+        return '%s:%s:%s' % (channel_id, self.KEYSPACE, message_id)
```

Comment:
I'm wondering if `get_key` should just be implemented by sub-classes? The key returned seems use-case specific.